### ISSUE TITLE
[BUG] GraphQL: This query requests up to 506,650 possible nodes which exceeds the maximum limit of 500,000.

### ISSUE THREAD

**Issue by sdwheeler** [dlvhdr/gh-dash#342]
**Describe the bug**
Receiving an error at startup. Also, the error display causes the layout in the terminal to be corrupted.

The log seems to show that the error occurs when querying the PRs for the MicrosoftDocs/PowerShell-Docs-DSC repo. That repo only has a total of 246 issues and PRs.

**To Reproduce**
Start dash with my configuration. The problem is intermittent.

**Screenshots**
I will add a screenshot when the error occurs again.

**Desktop (please complete the following information):**
- OS: Windows 11
- gh-dash version 3.14.0
  commit: bf3764ced0f35251bc315e634366ea4582313e90
  built at: 2024-02-17T17:02:04Z
  built by: goreleaser
  goos: windows
  goarch: amd64
- [debug.zip](https://github.com/dlvhdr/gh-dash/files/14450081/debug.zip) - contains the debug log and my configuration file

**Comment by sdwheeler** [comment]
Here is a screenshot showing the error and how it breaks the layout in the terminal.
![image](https://github.com/dlvhdr/gh-dash/assets/19415881/3e9a97db-4cae-4b0a-bc02-c37ddf2d2802)

Interesting to note, I only see this error on screen when I run dash without the `--debug` switch. If I use `--debug`, the error is in the error log but not on screen.

**Comment by KyleKing** [comment]
I'm also having the same issues with both `v3.14.0` and HEAD, but not when downgrading to `v3.13.1`:

```sh
gh extension install dlvhdr/gh-dash --pin='v3.13.1' --force
```

**Comment by dlvhdr** [comment]
You can use HEAD until I release a new version, it should fix the problem.
For anyone that doesn't want to use HEAD, you can set a lower limit in your config:
```yaml
prSections:
  - title: My PRs
    layout:
      lines:
        hidden: true
    filters: author:@me
    limit: 5
```

**Comment by KyleKing** [comment]
Thanks!!

On the workaround, shouldn't `default.prsLimit` be used when set?
https://github.com/KyleKing/dotfiles/blob/67371a68ccc550a1b336bd4d6f1484ee1ee7841d/dot_config/gh-dash/config.yml#L33

**Comment by dlvhdr** [comment]
Yeah it should fallback to that of the section doesn't define one. I guess even with 50 prs, I fetch too many nodes (a pr includes comments, review threads and a whole lot more nodes)
